android/engine/src/test/java/org/smartregister/fhircore/engine/configuration/view/ImmunizationProfileViewConfigurationTest.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2021 Ona Systems, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.smartregister.fhircore.engine.configuration.view
+
+import org.junit.Assert
+import org.junit.Test
+
+class ImmunizationProfileViewConfigurationTest {
+
+  @Test
+  fun immunizationProfileViewConfiguration() {
+    val immunizationProfileViewConfiguration = ImmunizationProfileViewConfiguration("anc", "app")
+
+    Assert.assertNotNull(immunizationProfileViewConfiguration)
+  }
+
+  @Test
+  fun immunizationProfileViewConfigurationsOfTest() {
+    val immunizationProfileViewConfigurationsOf = immunizationProfileViewConfigurationsOf()
+    Assert.assertNotNull(immunizationProfileViewConfigurationsOf)
```

Comment:
Perform assertions on the configuration properties to capture any changes made on the data class

Reply:
@ellykits changes addresses